### DISCUSSION TITLE
Support for swisstransfer.com?

### DISCUSSION THREAD

**Discussion by greigm** [pyload/pyload#4344] [Help] [answered]
The swisstransfer.com site seems commonly (though recently) popular among the taper crowd (jam bands etc.) but pyload doesn't seem to be able to decode the links and download the actual file(s) referenced.

eg. https://www.swisstransfer.com/d/04334a19-3cd1-40f4-97ba-897490e0fbce (apologies, if anyone is looking at that after 2023-08-30 as ST links auto-delete).

I've searched for mention of pyload and swisstransfer but turned up nothing. I would be happy to use this as a learning experience to help solve the problem, but I'm a bit of a python n00b, so a pointer in the right direction in terms of why it doesn't work, how it differs from other similar link-download sites, and steps to fix it would be great. Or, just give a man a fish, but anything is welcome.

Thanks!

EDIT: I've tried looking at the pixeldrain plugin for inspiration, but it uses a published api, which (as far as I can tell) swisstransfer doesn't have. This makes predicting the filenames/download hosts seem impossible. I hope I'm wrong!

**Comment by GammaC0de** [comment, marked as answer]
#4348
